tests/Faker.Tests/Common/NameTests.cs

```diff
@@ -25,5 +25,19 @@ public void Should_Get_Suffix()
 
 			Assert.That(new[] { suffix }, Is.SubsetOf(possibleSuffixes));
 		}
+
+		[Test]
+		[Repeat(100000)]
+		public void Compute_Time_Elapsed_By_The_First_Method()
+		{
+			Name.First();
```

Comment:
That is a huge performance gain, I'm not really surprised though.
If I remember correctly resx resource reading are notoriously slow